Properties/Strings.fr.resx

```diff
@@ -205,7 +217,7 @@
     <value>Notes</value>
   </data>
   <data name="Freshwater Fish" xml:space="preserve">
-    <value>Poisson d’eau douce</value>
+    <value>Poisson d\'eau douce</value>

```

Comment:
XML says that only characters that need to be escaped are `&` and `<`

So this shouldn't need to be escaped.